doc/source/.res/replace.txt

```diff
@@ -19,4 +19,6 @@
 .. |centos| replace:: CentOS
 .. |rhel| replace:: RHEL
 
-		      
+.. |version.next| replace:: 9.0
```

Comment:
We don't actually know the next major version will be called 9.0, however that is our suspicion based on current conversations.